cypress/e2e/awx/administration/topology-view.cy.ts

```diff
@@ -54,7 +63,7 @@ describe('Topology view', () => {
       });
   });
 
-  it('navigate to instance group detail when instance group is clicked from sidebar', () => {
+  it.skip('navigate to instance group detail when instance group is clicked from sidebar', () => {
```

Comment:
I see that you have skipped this test- It looks like it would be a better component test. Perhaps it can be moved to the component test file for this feature, and deleted from this file?